composer.json

```
@@ -11,19 +11,19 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8",
+        "php": "^7.3|^8.0",
         "ext-simplexml": "*",
-        "illuminate/config": "^6.0 || ^8.0",
-        "illuminate/container": "^6.0 || ^8.0",
-        "illuminate/contracts": "^6.0 || ^8.0",
-        "illuminate/database": "^6.0 || ^8.0",
-        "illuminate/events": "^6.0 || ^8.0",
-        "illuminate/http": "^6.0 || ^8.0",
-        "illuminate/routing": "^6.0 || ^8.0",
-        "illuminate/support": "^6.0 || ^8.0",
-        "illuminate/view": "^6.0 || ^8.0",
+        "illuminate/config": "^6.0 || ^9.0",
```

This should include `^8.0`, here and below.

```suggestion
        "illuminate/config": "^6.0 || ^8.0 || ^9.0",
```

---

right!